test2.txt

```diff
@@ -0,0 +1,16 @@
+Just some ipsm lorem stuff
+for a few lines to
+make a change
+big enough to
+surpass diff
+basic line
```

Comment:
[On the Git mailing list](https://github.com/webstech/gggmail3aea85a5-b8bb-f6af-cfe0-d11319b64232@example.com), Code Bot wrote ([reply to this](https://github.com/gitgitgadget/gitgitgadget/wiki/ReplyToThis)):

``````````email
This email is being reviewed.

   From: "Chris. Webster" <chris@webstech.net>
   
   2022-12-25 15:59:16.41
   
   test2.txt
   
   Signed-off-by: Chris. Webster <chris@webstech.net>
   ---
    test2.txt | 8 ++++++++
    1 file changed, 8 insertions(+)
   
   diff --git a/test2.txt b/test2.txt
   index fa0e7b5..f56d5d0 100644
   --- a/test2.txt
   +++ b/test2.txt
   @@ -6,3 +6,11 @@ surpass diff
    basic line
    limits.
    15:59:10.93  
   +Just some ipsm lorem stuff
   +for a few lines to
   +make a change
   +big enough to
   +surpass diff
   +basic line
   +limits.
   +15:59:16.40  
   -- 
   webstech
   
``````````